### PR TITLE
OSDOCS-5987-nwt2:adds nwt and API server bugs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-5987

Link to docs preview:
https://66743--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes:~:text=Previously%2C%20transmission

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1435,17 +1435,18 @@ This caused the Ingress Operator to report the DNS status as `ready` when there
 
 * Previously, the Ingress Operator was leaving the "spec.internalTrafficPolicy", "spec.ipFamilies", and "spec.ipFamilyPolicy" fields unspecified for `NodePort` and `ClusterIP` type services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by API default values. (link:https://issues.redhat.com/browse/OCPBUGS-13190[*OCPBUGS-13190*])
 
-* For {product-title} 4.14.0 and future versions, global IP forwarding is disabled by default on OVN-Kubernetes based cluster deployments.
+* Previously, transmission control protocol (TCP) connections were load balanced across all DNS. With this update, TCP connections are enabled to prefer local DNS endpoints. (link:https://issues.redhat.com/browse/OCPBUGS-9985[*OCPBUGS-9985*])
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:1438]:
```suggestion
* Previously, transmission control protocol (TCP) connections were load balanced for all DNS. With this update, TCP connections are enabled to prefer local DNS endpoints. (link:https://issues.redhat.com/browse/OCPBUGS-9985[*OCPBUGS-9985*])
``` 
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -484,7 +484,7 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-se
 [id="ocp-4-14-egress-ips-additional-networks"]
 ==== Support for egress IPs on additional network interfaces (Technology Preview)
 
-Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements. 
+Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements.
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:487]:
```suggestion
Support for egress IP addresses on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:1438]: @tmalove good catch.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:487]: I'll grab this, but this was just eliminating the trailing whitespace.

**Pull request merged**